### PR TITLE
Transak integration

### PR DESCRIPTION
**Pull request by Pedro-vk**
### Description

_Integrate Transak_

- Closes: #27 

### Tested

_Tested on Android and Transak staging_

### CODE REVIEW

**Review by gnardini — COMMENTED** [review]
Code looks good, on iOS the UX is a bit weird, you get directed to a screen but then a popup shows up (see video) but I guess it's not that big of a deal.

https://user-images.githubusercontent.com/6062888/109062603-e5722f80-76c6-11eb-880a-7e56f0ed8cb6.mov

Also, the redirectURL didn't work for me :/ I just got redirected back to the first screen of the experience:

Screen Shot 2021-02-24 at 17 41 49

Update: Per @tarikbellamine request I tried using a normal webpage as `redirectURL` and that worked perfectly, so it's probably an issue with the deep link. I'm not sure if there's a way to intercept link opens with this in-app browser, but that'd be an option is it's possible.

File: packages/mobile/src/fiatExchanges/TransakScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      &disableWalletAddressForm=true
+      &cryptoCurrencyCode=${asset}
+      &fiatCurrency=${currencyCode}
+      &defaultFiatAmount=${localAmount || minTxAmount}
+      &redirectURL=${encodeURIComponent(webRedirectUrl)}
+      &hideMenu=true
+    `.replace(/\s+/g, '')
+
+  const onNavigationStateChange = ({ url }: any) =>
+    !url.startsWith(webRedirectUrl) && navigateHome()
```
Comment (gnardini) [inline packages/mobile/src/fiatExchanges/TransakScreen.tsx:65]: The condition here is reversed, we need to navigate home if the url does start with the `webRedirectUrl`

**Review by tarikbellamine — APPROVED** [review]

**Comment by Celoqa** [comment]
Hi @i1skn we have verified this task using latest Android Internal Build V1.12.0(1004294340) & Test flight build V1.12.0(49) and updated the status in below sheet.
**Sheet:** https://docs.google.com/spreadsheets/d/1phHo7MUMQNLjuJOic1TlZwoHM2T0iecT07VLoKtiqMg/edit#gid=4156461 

**Observations:** 
- Warn popup is not shown when user tap on Transak button 
- Clabs production API keys is not active" popup is shown every time when user tap on Transak button.[Attachment](https://drive.google.com/file/d/1hAxeIHKg5y8i8e6NuCwwxYomtmcdGNb8/view?usp=sharing) 

**Note:** Can you please provide any BTC wallet address so, we can continue further testing in this task as we dont have any credentials for BTC wallet.

**Comment by i1skn** [comment]
@Pedro-vk can you please take a look on the comment above? Thanks!

**Comment by Pedro-vk** [comment]
@Celoqa I've requested access to the documents that you attached.

**Comment by Celoqa** [comment]
@Pedro-vk Sorry for the inconvenience i have given access to you can you please check it now.

**Comment by tarikbellamine** [comment]
Hi @Celoqa! There is no need to test Transak at this moment. We have an issue with their API key so we will be disabling this provider (see [this PR](https://github.com/celo-org/wallet/pull/145) where I disabled it).

In the next release, the way to test this is to make sure that it is loading properly.

**Comment by Celoqa** [comment]
Hi @tarikbellamine Thanks for confirmation sure we will test in next release that loading properly.

**Comment by Celoqa** [comment]
Hey @tarikbellamine FYI: We are not able to see Transak on latest released Android Internal Build V1.12.1(1004294341) & Test flight build V1.12.1(50). Can you please let us know have you removed this feature on latest Builds  ?
**Attachment:** [TransakNotDisplaying](https://drive.google.com/file/d/10V_UO-UmzvxF0QDFlrgFEdSfe59hiv5o/view?usp=sharing)
**Verified on Devices:** Samsung Galaxy A5(7.0), iPhone XR (14.4.1)

**Comment by gnardini** [comment]
@Celoqa We have removed this for 1.12, Transak will be live on 1.13.

**Comment by Celoqa** [comment]
Thanks @gnardini.

**Pull request merged**